sdk/nodejs/Makefile

```
@@ -112,6 +112,18 @@ ifneq ($(PULUMI_TEST_COVERAGE_PATH),)
 	if [ -e .nyc_output ]; then yarn run nyc report -r cobertura --report-dir $(PULUMI_TEST_COVERAGE_PATH); fi
 endif
 
+PULUMI_TEST_ORG ?= $(shell pulumi whoami --json | jq ".organizations[0]")
+TEST_GREP ?= .*
+
+# Watches all the typescript files and runs the tests every time they change.
+# Can optionally be given a regex of tests to match. For example, to iterate
+# on one test specifically, one could use the following command:
+#
+# $ make test_watch TEST_GREP="lists tag values"
+test_watch::
+	echo $(PULUMI_TEST_ORG)
```

This line is gonna be a bit weird when the user is not logged in.  I think I'd make it at least:

```suggestion
	echo PULUMI_TEST_ORG=$(PULUMI_TEST_ORG)
```

Or just leave it out completely, especially since it doesn't matter if `PULUMI_ACCESS_TOKEN` is not set, which I expect to be the case for a lot of developer environments.

---

Oops! This was actually a debugging thing I left in. I've just deleted it for now.